src/main/java/de/tum/in/www1/artemis/domain/submissionpolicy/LockRepositoryPolicy.java

```diff
@@ -5,15 +5,12 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 
-import de.tum.in.www1.artemis.domain.participation.Participation;
-import de.tum.in.www1.artemis.service.SubmissionPolicyService;
-
 /**
  * Configures a Lock Repository Policy.<br>
  * The Lock Repository Policy locks a participation repository after the participant submits
  * {@link SubmissionPolicy#submissionLimit} amount of times.<br>
  * The number of submissions in one participation is determined based on multiple factors.
- * More information on submission counts can be found at {@link SubmissionPolicyService#getParticipationSubmissionCount(Participation)}.
+ * More information on submission counts can be found at SubmissionPolicyService.getParticipationSubmissionCount(Participation).
```

Comment:
Why this change?

Reply:
Because the SubmissionPolicyService was imported, but now I remember that you can refer to the method without the need to import but by specifying the class with the package, so I updated it